app/src/main/java/com/example/qrchive/Fragments/ScanResultPopupFragment.java

```diff
@@ -61,8 +62,14 @@ public void onClick(DialogInterface dialog, int which, boolean isChecked) {
         builder.setPositiveButton("submit", new DialogInterface.OnClickListener() {
             @Override
             public void onClick(DialogInterface dialog, int which) {
-                getParentFragmentManager().beginTransaction().replace(R.id.fragment_container,new CaptureFragment(fbw),null).commit();
-                //sendResult();
```

Comment:
Because this line was commented out, the Firebase stuff was never gonna run any way